src/hooks/useSwap.ts

```diff
@@ -77,15 +74,6 @@ export const useSwap = ({ accountId, selector }: Props) => {
   const { getNearBlock } = useNearBlock()
   const { getTransactionScan } = useTransactionScan()
 
```

Comment:
What is the point of making hooks like `useNearBlock` or `useTransactionScan`. They have no internal state or effects and create a function on every component re-render.

Why not just import them as  a function to a component from now on ?